test/tests/val-2b/comparison_val-2b.py

```diff
@@ -4,13 +4,17 @@
 from matplotlib import gridspec
 import pandas as pd
 from scipy import special
+import os
+import git
 
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
 
 fig = plt.figure(figsize=[6.5, 5.5])
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-expt_data = pd.read_csv("./gold/experimental_data.csv")
+expt_data = pd.read_csv(os.path.join(git.Repo('.',search_parent_directories=True).working_tree_dir, "test/tests/val-2b/gold/experimental_data.csv"))
```

Comment:
The usage of `git.Repo` here is the root of the failure in Apptainer that necessitated use of `heavy` as a "workaround". This is because Apptainer installs a copy of TMAP8 and its tests, and then tests them. At that stage, it is no longer in a git repository, but on the regular apptainer file system. Hence, an `InvalidGitRepositoryError` is raised.

The use of `os.chdir(os.path.dirname(__file__))` on Line 11 of this file, which sets the working directory to the script directory, makes the directory references in the code more robust by setting the working directory to a standard location. So, all directory references can simply be relative to the working directory. You should just be able to say `./gold/experimental_data.csv` here when reading the CSV data. Why are you using `git.Repo` to do that? Was `chdir` not working correctly in your testing?

Reply:
Thank you so much @cticenhour!
My guess is that @lin-yang-ly was doing so because he used ver-1a as an example, and it was using:
```
tmap8_prediction = pd.read_csv(os.path.join(git.Repo('.',search_parent_directories=True).working_tree_dir, "test/tests/ver-1a/gold/ver-1a_csv.csv"))
```

@lin-yang-ly, could you use @cticenhour's recommendation and let us know if that works? And if it does, update ver-1a as well. 
Thank you!

Reply:
Yes, give it a try - I made that script when we weren't using Apptainer and installing TMAP8 on every PR.

Reply:
Thank you, @cticenhour and @simopier! I will try to use the relative path based on the script directory.

Reply:
It works with the updated relative path at #175! I will continue to update all cases in this PR.

Also, due to we call the python script from two directories (test folder and documentation folder), I use a "if" to distinguish the two working directories. Just let me know if you have better idea.
```
# Changes working directory to script directory (for consistent MooseDocs usage)
script_folder = os.path.dirname(__file__)
os.chdir(script_folder)

if "/TMAP8/doc/" in script_folder:     # if in documentation folder
    csv_folder = "../../../../test/tests/ver-1fa/gold/ver-1fa_csv_line_0011.csv"
else:                                  # if in test folder
    csv_folder = "./gold/ver-1fa_csv_line_0011.csv"
```

Reply:
That works for me, thank you!